app/models/manageiq/providers/nsxt/nsxt_client.rb

```diff
@@ -15,64 +15,68 @@ def initialize(server, user, password, verify_ssl = false)
   end
 
   def get_tier_1(id)
-    get("infra/tier-1s/#{id}")
+    get("tier-1s/#{id}")
   end
 
   def get_tier_1s
-    list('infra/tier-1s')
+    list('tier-1s')
   end
 
   def get_segment(id)
-    get("infra/segments/#{id}")
+    get("segments/#{id}")
   end
 
   def get_segments
-    list('infra/segments')
+    list('segments')
   end
 
   def get_security_policy(id)
-    get("infra/domains/default/security-policies/#{id}")
+    get("domains/default/security-policies/#{id}")
   end
 
   def get_security_policies
-    list('infra/domains/default/security-policies')
+    list('domains/default/security-policies')
   end
 
   def get_security_policy_rules(id)
-    list("infra/domains/default/security-policies/#{id}/rules")
+    list("domains/default/security-policies/#{id}/rules")
   end
 
   def get_service
-    get("infra/services/#{id}")
+    get("services/#{id}")
   end
 
   def get_services
-    list('infra/services')
+    list('services')
   end
 
   def get_group(id)
-    get("infra/domains/default/groups/#{id}")
+    get("domains/default/groups/#{id}")
   end
 
   def get_groups
-    list('infra/domains/default/groups')
-  end
-
-  def get_group_members(id)
-    list("infra/domains/default/groups/#{id}/members/virtual-machines")
+    list('domains/default/groups')
   end
 
   private
 
   def list(url)
     json = get(url)
-    return json['results'] || []
+    json.nil? || json['results'].nil? ? [] : json['results']
   end
 
   def get(url)
-    response = @client.get("#{@server}/policy/api/v1/#{url}")
-    return nil if response.body.empty? || response.code != 200
+    response = @client.get("#{@base_url}/#{url}")
+    if response.body.empty? || response.code != 200
+      $nsxt_log.warn("Invalid response #{response} for REST call #{url}")
+      return nil
+    end
     json = JSON.parse(response.body)
     return json
   end
-end
\ No newline at end of file
+
+  API_VERSIONS = {
```

Comment:
I think this would more accurately be called the path component, not just the api_version

Reply:
How would you name it?

Reply:
Constant API_VERSIONS is not needed anymore.